WordPress/Classes/ViewRelated/Reader/ReaderTagStreamHeader.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+@objc public class ReaderTagStreamHeader: UIView, ReaderStreamHeader
+{
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var followButton: PostMetaButton!
+    public var delegate: ReaderStreamHeaderDelegate?
+
+
+    // MARK: - Lifecycle Methods
+
+    public override func awakeFromNib() {
+        super.awakeFromNib()
+
+        applyStyles()
+    }
+
+    func applyStyles() {
+        WPStyleGuide.applyReaderStreamHeaderTitleStyle(titleLabel)
+    }
+    
+    
+    // MARK: - Configuration
+
+    public func configureHeader(topic: ReaderTopic) {
+        titleLabel.text = topic.title
+        if topic.isSubscribed {
+            WPStyleGuide.applyReaderStreamHeaderFollowingStyle(followButton)
+        } else {
+            WPStyleGuide.applyReaderStreamHeaderNotFollowingStyle(followButton)
+        }
+    }
+
+
+    // MARK: - Actions
+
+    @IBAction func didTapFollowButton(sender: UIButton) {
+        delegate!.handleFollowActionForHeader(self)
```

Comment:
(Sorry about being annoying!). Just in case, maybe, `delegate?.` instead of `!`. For peace of mind! :smile: